drivers/health_comprehensive_assessment/app/views/health_comprehensive_assessment/assessments/_form.haml

```diff
@@ -0,0 +1,165 @@
+%h1 Comprehensive Assessment
```

Comment:
I'm probably missing it somewhere, but I'm not seeing the navigation for this.  Feels like we should definitely have section navigation on the left.